tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include "tls/s2n_tls.h"
+/* To test static functions */
+#include "tls/s2n_server_new_session_ticket.c"
+
+#define ONE_HOUR_IN_NANOS   3600000000000
+#define TWO_HOURS_IN_NANOS  ONE_HOUR_IN_NANOS * 2
+
+int main(int argc, char **argv)
+{
```

Comment:
For anything you annotated for the compliance tool, ideally you would also annotate the corresponding test (but with type=test before the quote).